consensus/src/experimental/buffer_manager.rs

```diff
@@ -251,20 +254,25 @@ impl BufferManager {
         unreachable!("Aggregated item not found in the list");
     }
 
-    /// It pops everything in the buffer and if reconfig flag is set, it stops the main loop
-    async fn process_reset_request(&mut self, request: ResetRequest) {
-        let ResetRequest { tx, stop } = request;
-        debug!("Receive reset");
-
-        self.stop = stop;
+    async fn reset(&mut self) {
```

Comment:
can you add the documentation for this method, as name is pretty generic and it has important semantics